components/dashboard/src/projects/ProjectListItem.tsx

```diff
@@ -25,6 +25,12 @@ export const ProjectListItem: FunctionComponent<ProjectListItemProps> = ({ proje
     const [showRemoveModal, setShowRemoveModal] = useState(false);
     const { data: prebuild, isLoading } = useLatestProjectPrebuildQuery({ projectId: project.id });
 
+    const enablePrebuilds =
+        !!project.settings?.enablePrebuilds ||
+        // TODO(at): out of scope for now, but once we've migrated the settings of existings projects
+        // we can remove the implicit enablement here
+        !project.settings;
```

Comment:
would it make sense to move this logic to a more central place, i.e. `ProjectSettings.isPrebuildsEnabled`?

Reply:
Yes, that makes sense. I'll add this. You'll find a similar expression in prebuild-manager.ts.

Reply:
(non-blocking) Following up here: Was [this method](https://github.com/gitpod-io/gitpod/blob/0492cef1498661962d160bbc17cb1da08837fc09/components/gitpod-protocol/src/teams-projects-protocol.ts#L66) meant to be used here?